inc/class-storefront.php

```diff
@@ -155,7 +155,7 @@ public function widgets_init() {
         $widget_tags = array(
           'before_widget' => '<div id="%1$s" class="widget %2$s">',
           'after_widget'  => '</div>',
-          'before_title'  => '<span class="widget-title">',
+          'before_title'  => '<span class="h3 widget-title">',
```

Comment:
Hmm...
After that I did point at Yoast website using `<span>` as widget titles, you said:

> For the headings, let's go with a div and make the args filterable so folks can easily change if they want.

Did I miss something ?